src/main/java/org/osc/sdk/controller/element/InspectionPortElement.java

```diff
@@ -23,7 +23,7 @@
  * If the ingress and egress ports are the same, then the same port will need to be returned for either of the methods.
  */
 @ConsumerType
-public interface InspectionPortElement {
+public interface InspectionPortElement extends Element {
```

Comment:
We shouldnt need to, the javadoc will already have a link to the interface this one extends from. Adding more info to that will be redundant.